src/System.Diagnostics.Process/src/System.Diagnostics.Process.builds

```diff
@@ -11,6 +11,9 @@
     <Project Include="System.Diagnostics.Process.csproj">
       <OSGroup>Windows_NT</OSGroup>
     </Project>
+    <Project Include="System.Diagnostics.Process.csproj">
+      <TargetGroup>net461</TargetGroup>
+    </Project>
     <Project Include="facade\System.Diagnostics.Process.csproj">
```

Comment:
It would have the same outcome but just less churn in this project file. I can live with either approach actually but we should have both facades being generated from the same project.